.travis.yml

```diff
@@ -7,7 +7,6 @@ node_js:
 - '6.1'
 env:
   matrix:
-  - TEST_BROWSER_NAME=PhantomJS
```

Comment:
all sorts of pieces of javascript that are present in all modern browsers are missing from PhantomJS < 2.5 (which is in beta).  You can still run locally with Karma -- I've updated the config so that you can run against any of your installed browsers.